README.md

```diff
@@ -2,83 +2,83 @@
 
 ![Lint and Build](https://github.com/gordon-cs/gordon-360-ui/workflows/Lint%20and%20Build/badge.svg)
 
-This project is the frontend of Gordon 360 in React. [The retired frontend](https://github.com/gordon-cs/Project-Bernard/) was built in EmberJS, but was deemed too slow and difficult to maintain. To see the specifics of the rebuild, see [`ROADMAP.md`](https://github.com/gordon-cs/gordon-360-ui/blob/develop/old_documentation/ROADMAP.md).
+The frontend of [Gordon 360](https://360.gordon.edu). This project is the User interface for Gordon College's student portal. It is written in [React](https://reactjs.org/) and connects to the [Gordon 360 backend](https://github.com/gordon-cs/gordon-360-api).
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app). Read the user guide [here](https://github.com/facebook/create-react-app/blob/master/packages/cra-template/template/README.md).
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
 ## Contents
 
-- [Server Notes](#server-notes)
-- [Libraries](#libraries)
+- [Getting Started](#getting-started)
+- [Connecting to the Backend](#connecting-to-the-backend)
```

Comment:
Why are you including something that is a part of local development here? I thought we were moving all of that to `Local Development.md`

Reply:
I think we miscommunicated yesterday. The instructions for developing locally are the primary thing that most developers are looking for in the README, so I think that should be the first thing in the README. Also, those instructions ended up being fairly concise after I re-wrote them. I removed `Local Development.md` in favor of that.